dotcom-rendering/src/web/browser/islands/onInteraction.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Use this function to delay execution of something until an element is interacted
+ * with
+ *
+ * @param element : The html element that we want to wait for an interaction on;
+ * @param callback : This is fired when the element is clicked on
+ */
+export const onInteraction = (
+	element: HTMLElement,
+	callback: (e: HTMLElement) => void,
+): void => {
+	element.addEventListener(
+		'click',
```

Comment:
Does the click event also get fired if I press the enter key when an element has focus?

Reply:
There’s a demo video suggesting that it does!